public/app/features/datasources/components/picker/DataSourceModal.tsx

```diff
@@ -159,7 +160,7 @@ export function DataSourceModal({
           className={styles.searchInput}
           value={search}
           prefix={<Icon name="search" />}
-          placeholder="Search data source"
+          placeholder={t('data-source-picker.input-placeholder', 'Select data source')}
```

Comment:
```suggestion
          placeholder={t('data-source-picker.modal.input-placeholder', 'Select data source')}
```

Reply:
That makes sense 👍,  I've made the changes.